lib/fathead/sass/output.txt

```diff
@@ -0,0 +1,159 @@
+:compressed	A										Compressed style takes up the minimum amount of space possible, having no whitespace except that necessary to separate selectors and a newline at the end of the file. It also includes some other minor compressions, such as choosing the smallest representation for colors. It’s not meant to be human-readable. For example:<pre class="code ruby"><code class="ruby">#main{color:#fff;background-color:#000}#main p{width:10em}.huge{font-size:10em;font-weight:bold;text-decoration:underline}</code></pre>	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#_16
+Extending Sass	A										Sass provides a number of advanced customizations for users with unique requirements. Using these features requires a strong understanding of Ruby.	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#extending_sass
+Defining Custom Sass Functions	A										Users can define their own Sass functions using the Ruby API. For more information, see the source documentation.	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#defining_custom_sass_functions
+Cache Stores	A										Sass caches parsed documents so that they can be reused without parsing them again unless they have changed. By default, Sass will write these cache files to a location on the filesystem indicated by :cache_location. If you cannot write to the filesystem or need to share cache across ruby processes or machines, then you can define your own cache store and set the:cache_store option. For details on creating your own cache store, please see the source documentation.	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#cache_stores
+Custom Importers	A										Sass importers are in charge of taking paths passed to @import and finding the appropriate Sass code for those paths. By default, this code is loaded from the filesystem, but importers could be added to load from a database, over HTTP, or use a different file naming scheme than what Sass expects.	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#custom_importers
+rgb	A										<pre><code>rgb($red, $green, $blue)</code></pre><br>Creates a Color from red, green, and blue values.	http://sass-lang.com/documentation/Sass/Script/Functions.html#rgb-instance_method
+rgba	A										<pre><code>rgba($red, $green, $blue, $alpha)</code></pre><br>Creates a Color from red, green, blue, and alpha values.	http://sass-lang.com/documentation/Sass/Script/Functions.html#rgba-instance_method
```

Comment:
It looks like `rgba()` is an interesting edge case. It has two syntaxes:

- `rgba($color, $alpha)`
- `rgba($red, $green, $blue, $alpha)`

Do you happen to know if there are more like this?

I think in this case we'll want to indicate both usages in a single article as we can't have two different articles with identical titles.